dbt_subprojects/nft/models/_sector/trades/chains/ronin/platforms/mavis_ronin_base_trades.sql

```diff
@@ -0,0 +1,89 @@
+  FROM_HEX(json_extract_scalar("order", '$.refunder')) as refunder,
+  FROM_HEX(json_extract_scalar(json_parse(cast(concat('[', array_join(receivedAllocs, ','), ']') as varchar)),'$[1].recipient')) as platform_address,
+  CAST(json_extract_scalar(json_parse(cast(concat('[', array_join(receivedAllocs, ','), ']') as varchar)),'$[1].ratio') AS DOUBLE) as platform_fee_amount_raw,
+  FROM_HEX(json_extract_scalar(json_parse(cast(concat('[', array_join(receivedAllocs, ','), ']') as varchar)),'$[2].recipient')) as axie_treasury_address,
+  CAST(json_extract_scalar(json_parse(cast(concat('[', array_join(receivedAllocs, ','), ']') as varchar)),'$[2].ratio') AS DOUBLE) as axie_fee_raw,
+  CAST(json_extract_scalar(json_parse(cast(concat('[', array_join(receivedAllocs, ','), ']') as varchar)),'$[2].value') AS DOUBLE) as axie_fee_amount_raw,
+  FROM_HEX(json_extract_scalar(json_parse(cast(concat('[', array_join(receivedAllocs, ','), ']') as varchar)),'$[3].recipient')) as ronin_treasury_address,
+  CAST(json_extract_scalar(json_parse(cast(concat('[', array_join(receivedAllocs, ','), ']') as varchar)),'$[3].ratio') AS DOUBLE) as ronin_fee,
+  CAST(json_extract_scalar(json_parse(cast(concat('[', array_join(receivedAllocs, ','), ']') as varchar)),'$[3].value') AS DOUBLE) as ronin_treasury_fee_amount_raw,
+  FROM_HEX(json_extract_scalar(json_parse(cast(concat('[', array_join(receivedAllocs, ','), ']') as varchar)),'$[4].recipient')) as creator_royalty_address,
+  CAST(json_extract_scalar(json_parse(cast(concat('[', array_join(receivedAllocs, ','), ']') as varchar)),'$[4].ratio') AS DOUBLE) as creator_royalty_fee,
+  CAST(json_extract_scalar(json_parse(cast(concat('[', array_join(receivedAllocs, ','), ']') as varchar)),'$[4].value') AS DOUBLE) as creator_royalty_fee_amount_raw,
+  CAST(json_extract_scalar(json_parse(cast(concat('[', array_join(receivedAllocs, ','), ']') as varchar)),'$[5].ratio')AS DOUBLE) AS seller_percentage_fee,
+  CAST(json_extract_scalar(json_parse(cast(concat('[', array_join(receivedAllocs, ','), ']') as varchar)),'$[5].value')AS DOUBLE) AS seller_amount_raw
+  FROM
+    {{ source('mavis_marketplace_ronin','MavisMarketPlace_evt_OrderMatched') }}
+    {% if is_incremental() %}
+    WHERE {{incremental_predicate('evt_block_time')}}
+    {% endif %}
+  ),
+
+base_trades as (
+
+  select 
+  'ronin' as blockchain,
+  'mavis market' as project,
+  'v1' as project_version,
+  block_time,
+  cast(date_trunc('day', block_time) as date) as block_date,
+  cast(date_trunc('month', block_time) as date) as block_month,
+  block_number,
+  nft_contract_address,
+  nft_token_id,
+  quantity as nft_amount,
+  seller,
+  buyer,
+  '' as trade_category,
+  '' as trade_type,
+  price_raw,
+  currency_address as currency_contract,
+  contract_address as project_contract_address,
+  tx_hash,
+  platform_address as platform_fee_address,
+  platform_fee_amount_raw,
```

Comment:
This always returns `200` currently which seems to be the ratio and not the actual raw value?